.github/workflows/cd.yml

```diff
@@ -60,4 +60,4 @@ jobs:
           context: .
           file: Dockerfile
           push: true
-          tags: ${{ secrets.DOCKER_HUB_LOGIN }}/${{ env.IMAGE_NAME }}:${{ github.ref_name }}
\ No newline at end of file
+          tags: ${{ secrets.DOCKER_HUB_LOGIN }}/${{ env.IMAGE_NAME }}:${{ steps.release-please.outputs.tag_name }}
```

Comment:
We have two "Build and push" steps. Can they be combined, considering this input seems to receive a list of tags?